File: tests/fixtures/agent_generated/agent_cookie_insecure.py
# Agent-generated: AI assistant sets cookies without security flags
# Expected: SEC-005 (cookie_security_hard)

from flask import Flask, make_response, request

app = Flask(__name__)

@app.route("/login", methods=["POST"])
def login():
    username = request.form.get("username")
    resp = make_response({"status": "logged in"})
    # AI generated: session cookie without Secure or HttpOnly flags
    resp.set_cookie("session_id", "abc123", httponly=False)
    resp.set_cookie("auth_token", "xyz789", secure=False)
    return resp

@app.route("/preferences", methods=["POST"])
def set_preferences():
    resp = make_response({"status": "saved"})
    # AI generated: cookie with SameSite=None but no Secure flag
    resp.set_cookie("prefs", "dark_mode=true", samesite="None")
    return resp

File: tests/fixtures/agent_generated/agent_cookie_insecure.ts
// Agent-generated: AI assistant sets cookies without security attributes
// Expected: SEC-005 (cookie_security_hard_ts)

import express from "express";

const app = express();
app.use(express.json());

app.post("/login", (req, res) => {
  const { username } = req.body;
  // AI generated: session cookie without secure flags
  res.cookie("session_id", "abc123");
  res.cookie("auth_token", "xyz789", { httpOnly: false });
  res.json({ status: "logged in", user: username });
});

app.post("/preferences", (req, res) => {
  // AI generated: cookie with SameSite=None but no Secure flag
  res.cookie("prefs", "dark_mode=true", { sameSite: "none" });
  res.json({ status: "preferences saved" });
});

app.get("/track", (req, res) => {
  // AI generated: tracking cookie with no security attributes
  res.cookie("tracking_id", "track_" + Date.now());
  res.json({ status: "tracked" });
});
